content/language-agnostic/encyclopedia/what-is-a-terminal.md

```diff
@@ -0,0 +1,23 @@
+<!--
+---
+Subjects:
+  - "What is a Terminal?"
+Tags: 
```

Comment:
Should be from https://github.com/Codecademy/codepedia-content/blob/main/documentation/tags.md